### ISSUE TITLE
Icon Request: icon-resize-topleft / icon-resize-topright / icon-resize-nw / ...

### ISSUE THREAD

**Issue by roberthartung** [FortAwesome/Font-Awesome#1332]
I would like to request two more resize icons.

Currently there are only 'icon-resize-horizontal' and 'icon-resize-vertical' but I'd appreciate if there are two additional ones:

icon-resize-topleft (aliases: icon-resize-north-west // icon-resize-south-east // icon-resize-nw // icon-resize-se)

icon-resize-topright (icon-resize-north-east // icon-resize-south-west // icon-resize-ne // icon-resize-sw).

So basically resize icons that are rotated by 45 degree.

Those could very useful for various web applications. including mine.

Regards
Robert

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’re closing this issue because it already exists in Font Awesome Pro! [Take a look at it here.](https://fontawesome.com/icons/arrow-up-right-and-arrow-down-left-from-center?f=classic&s=solid)
